src/scripts/shards/fluteEffectRunner.ts

```diff
@@ -8,13 +8,13 @@ class fluteEffectRunner {
         GameHelper.enumStrings(GameConstants.FluteItemType).forEach((itemName) => {
             const item = (ItemList[itemName] as FluteItem);
             if (item.multiplierType) {
-                multiplier.addBonus(item.multiplierType, () => this.isActive(itemName)() ? 1 + ((item.multiplyBy - 1) * AchievementHandler.achievementBonus()) : 1);
+                multiplier.addBonus(item.multiplierType, () => this.isActive(itemName)() ? this.getFluteMultiplier(itemName) : 1);
```

Comment:
`getFluteMultiplier` does the `isActive` check and defaulting to 1 for you, so no need to do it again here.

Reply:
[190e4d0](https://github.com/pokeclicker/pokeclicker/pull/1791/commits/190e4d0d8790a11f51e11e6a5a1f40812cfe7a2a)